src/components/post/Content.jsx

```diff
@@ -4,11 +4,19 @@ import React from "react"
 import PropTypes from "prop-types"
 
 const MarkdownContent = ({ content }) => {
-  return (
-    <ReactMarkdown rehypePlugins={[rehypeRaw]}>
-      {content?.content?.content}
-    </ReactMarkdown>
-  )
+  if (content?.content?.content) {
+    return (
+      <ReactMarkdown rehypePlugins={[rehypeRaw]}>
+        {content?.content?.content}
+      </ReactMarkdown>
+    )
+  } else if (content?.subContent?.subContent) {
+    return (
+      <ReactMarkdown rehypePlugins={[rehypeRaw]}>
+        {content?.subContent?.subContent}
+      </ReactMarkdown>
+    )
+  }
```

Comment:
This can be something like `{A ? A : B ? B : ""}`. If nothing in markdown, it is just an empty div in the HTML.